docs/contribute/installation.md

```diff
@@ -96,6 +96,31 @@ Happens when you are on **Windows**. `climatemind-backend_db`  and `climatemind-
 
 That means you forgot to take the [required preparation ](installation.md#before-you-start)step. Please remove the `climatemind-backend` folder and start from scratch.
 
+#### \[SQL Server]Invalid object name 'sessions'.
+
+Other symptoms: 
+* Webapp front page generates an error message: "Something went wrong initializing your session. :( Please refresh the page to try again." 
+* Postman API calls to http://localhost:5000/session return: "An error occurred while saving the session to the database."
+
+The database failed to migrate when the project was built and the schema is missing. Run the following commands from the `climatemind-backend` folder.
+
+First, remove broken the containers:
+```shell
+docker-compose -p climatemind-backend -f docker/docker-compose.yml down
```

Comment:
Do I understand correctly you didn't remove database volume with -v ?

Reply:
That's correct. The issue was resolved for me after bringing down the containers and rebuilding the project. Added removing the database as a final suggestion for others just in case.